tests/testthat/test-make_linter_from_xpath.R

```diff
@@ -30,3 +30,54 @@ test_that("input validation works", {
   err_msg <- "lint_message is required"
   expect_error(make_linter_from_xpath(""), err_msg, fixed = TRUE)
 })
+
+test_that("basic usage of make_linter_from_function_xpath works", {
+  linter <- make_linter_from_function_xpath(
+    "sum",
+    "following-sibling::SYMBOL_SUB[text() = 'na.rm']",
+    "na.rm"
+  )
+  expect_type(linter, "closure")
+  expect_no_lint("sum()", linter())
+  expect_lint("sum(na.rm=TRUE)", "na.rm", linter())
+})
+
+test_that("input validation works for make_linter_from_function_xpath", {
```

Comment:
Is it possible to use parameterized testing here?

Reply:
I'm not seeing a way to do so that would improve things, any suggestion?